WooCommerce/Classes/ViewRelated/Dashboard/Reviews/ReviewsDashboardCardViewModel.swift

```diff
@@ -147,8 +181,14 @@ private extension ReviewsDashboardCardViewModel {
         data = reviews
             .map { review in
                 let product = reviewProducts.first { $0.productID == review.productID }
-                // TODO: also fetch notification
-                return ReviewViewModel(review: review, product: product, notification: nil)
+                let notification = notifications.first { notification in
+                    if let notificationReviewID = notification.meta.identifier(forKey: .comment) {
```

Comment:
Same matching logic as in https://github.com/woocommerce/woocommerce-ios/blob/48d44a8560b6c6b8fa27f4f0fae0af83699dfa1b/WooCommerce/Classes/ViewRelated/Reviews/ReviewsDataSource.swift#L194